io/net/infiniband/ib_pingpong.py.data/ib_pingpong_roce.yaml

```diff
@@ -17,8 +17,8 @@ Test: !mux
         ext_test_opt: -s 524288,-s 524288 -n 10000,-s 524288 -n 100000,-q 32,-e -s 524288 -n 10000 -q 32
     xsrq_pingpong:
         tool: ibv_xsrq_pingpong
-        test_opt: -s 1024,basic,-n 100000,-e,-l 2,-m 1024
-        ext_test_opt: -s 524288,-s 524288 -n 10000,-s 524288 -n 100000,-e -s 524288 -n 10000
+        test_opt: -s 1024,basic,-n 8192,-e,-l 2,-m 1024
+        ext_test_opt: -s 524288,-s 524288 -n 8192,-s 524288 -n 8192,-e -s 524288 -n 8192
```

Comment:
@PavamanSubramaniyam was this failing before ?
any recent changes that causing this fail for beyond this value?

Reply:
@Naresh-ibm Yes, it was failing before. So, had reported a bug and the developer mentioned that for ibv_xsrq_pingpong utility the maximum allowed number of -n option value is 8192.
So, accordingly changing the same in the yaml file.